<launch>

    <!-- Default value is my folder with the videos -->

    <arg name="vid_folder" default="/home/k21082509/ros_ws/src/resit_coursework/Videos/cw_resit_videos" />

    <!-- For Testing -->
    <!-- <node name="rviz" pkg="rviz" type="rviz" args="-d $(find rosplan_stage_demo)/config/rosplan_stage_demo.rviz" /> -->

    <include file="$(find ros_vosk)/launch/ros_vosk.launch" />

    <node name="room2med_server" pkg="resit_coursework" type="room2med.py" output="screen" />

    <node name="yolo_ros" pkg="resit_coursework" type="yolo_ros.py" output="screen" />

    <node name="main_node" pkg="resit_coursework" type="main_node.py" output="screen" />

    <include file="$(find rosplan_stage_demo)/launch/empty_stage_single_robot.launch" />

    <node name="test_script" pkg="resit_coursework" type="itr_cwCS24_resit" output="screen" args="$(arg vid_folder)" />

</launch>
